galaxy_ng/app/constants.py

```diff
@@ -10,3 +10,8 @@ class CertificationStatus(enum.Enum):
 class DeploymentMode(enum.Enum):
     STANDALONE = 'standalone'
     INSIGHTS = 'insights'
+
+
+class UIAPIVersions(enum.Enum):
```

Comment:
It doesn't seem like valid Enum usage. Here enum is not a type that represents static set of values, but rather a namespace for different constants. Please remove this enum and replace it with constants.